Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/SeedingOrthogonalAlgorithm.hpp

```diff
@@ -62,6 +62,7 @@ class SeedingOrthogonalAlgorithm final : public BareAlgorithm {
 
  private:
   Config m_cfg;
+  Acts::SeedFinderOrthogonal<SimSpacePoint> m_finder;
```

Comment:
it works since this is the example and beam spot and mag field are constant, right? What if they change for every event?